### DISCUSSION TITLE
Memory full

### DISCUSSION THREAD

**Discussion by Rhoenit** [telekom-security/tpotce#1384] [General]
We were using an old version of the tpot, which was working perfectly fine. We replaced it with a new Honeypot running the last version in a VM. However, shortly after, we encountered an issue where the memory gets filled up quickly. the ddospot seems to be a major cintributor to this. How can we clear the database and disable the container?

![Hopo_Memory](https://github.com/telekom-security/tpotce/assets/78480832/d41f3523-0e55-482a-bedd-e9d9a064d6db)
![memoryddospot](https://github.com/telekom-security/tpotce/assets/78480832/07c71a76-ac99-4de5-b7cd-34ca8ca17723)

**Comment by t3chn0m4g3** [comment]
Please follow the T-Pot documentation on how to adjust the [tpot.yml](https://github.com/telekom-security/tpotce#adjust-tpotyml). It is a `docker-compose` file, the structure is documented [here](https://docs.docker.com/compose/compose-file/).
Basically you remove the `ddospot` service and its network references.
Afterwards you can remove the `ddospot` folder from the file system or simply delete the `.sqlite3` files to regain free space.

**Reply by Rhoenit** [reply]
We still have the same problem but now I can´t find tpot.yml anymore. What is the path in the new version?
